nomad-aws/examples/basic/main.tf

```diff
@@ -38,7 +38,7 @@ module "nomad-aws" {
 
   # Location of your Nomad server endpoint. This should be exposed from your
   # server installation via a load balancer service.
-  server_endpoint = "example.com:4647"
+  nomad_server_hostname = "example.com"
```

Comment:
the name `nomad_server_hostname` seems to suggest, this is different than the application DNS name. We should use - `server_hostname`